build.xml

```diff
@@ -96,6 +96,9 @@
                         <replacefilter token="@LOG_DIR@" property="user_input.LOG_DIR"/>
                         <replacefilter token="@MEMCACHE_HOST@" property="user_input.MEMCACHE_HOST"/>
                         <replacefilter token="@PLAY_SERVER_BASE_DIR@" property="user_input.PLAY_SERVER_BASE_DIR"/>
+						<replacefilter token="@CACHE_TYPE@" property="user_input.CACHE_TYPE"/>
+						<replacefilter token="@CLUSTER_NAME@" property="user_input.CLUSTER_NAME"/>
```

Comment:
the following two are optional - must make sure that if they are not filled we don't fail